MVCDemos/Views/Form/GroupedFields.cshtml

```diff
@@ -64,3 +69,10 @@
         .FormData(Model)
     )
 </div>
+<script>
```

Comment:
Please add a line break before the <script> tag, and remove any empty lines inside the <script> tag. For both approaches.